File: backend/enum-jpa-validation-app/src/main/java/com/example/enumapp/web/ApiMessageCodes.java
package com.example.enumapp.web;

public final class ApiMessageCodes {

    /** 요청 검증 실패 (400) */
    public static final String VALIDATION_ERROR = "VALIDATION_ERROR";
    /** 잘못된 파라미터/검색 조건 (400) */
    public static final String BAD_REQUEST = "BAD_REQUEST";
    /** 리소스 없음 (404) */
    public static final String ORDER_NOT_FOUND = "ORDER_NOT_FOUND";
    public static final String FILE_NOT_FOUND = "FILE_NOT_FOUND";
    /** 서버 내부 오류 (500) */
    public static final String INTERNAL_ERROR = "INTERNAL_ERROR";

    private ApiMessageCodes() {
    }
}

File: backend/enum-jpa-querydsl-kotlin-app/src/main/kotlin/com/example/enumapp/web/ApiMessageCodes.kt
package com.example.enumapp.web

object ApiMessageCodes {
    /** 요청 검증 실패 (400) */
    const val VALIDATION_ERROR = "VALIDATION_ERROR"
    /** 잘못된 파라미터/검색 조건 (400) */
    const val BAD_REQUEST = "BAD_REQUEST"
    /** 리소스 없음 (404) */
    const val ORDER_NOT_FOUND = "ORDER_NOT_FOUND"
    const val FILE_NOT_FOUND = "FILE_NOT_FOUND"
    /** 서버 내부 오류 (500) */
    const val INTERNAL_ERROR = "INTERNAL_ERROR"
}
